alluxio-ui/common/src/App.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+import React from 'react';
+import {LoadingMessage} from './components';
+
+class App extends React.Component {
+  public render() {
+    return (
+      <div className="App">
+        <header className="App-header">
+          <h1 className="App-title">Welcome to the Alluxio Shared Web Application</h1>
+        </header>
+        <p className="App-intro">
+          This will contain reusable components for other web applications.
+        </p>
+        <p>
+          Control list:
+          <LoadingMessage/>
+        </p>
+      </div>
+    );
+  }
+}
```

Comment:
Is this page and its contents necessary?

Reply:
The common app is technically a buildable runnable product, so if we want to keep it that way, then yes.